File: tests/test-minecraft-osp/src/main/java/pize/tests/voxelgame/client/block/vanilla/VoidAir.java
package pize.tests.voxelgame.client.block.vanilla;

import pize.tests.voxelgame.client.block.BlockProperties;
import pize.tests.voxelgame.client.resources.GameResources;

public class VoidAir extends BlockProperties{
    
    public VoidAir(int id){
        super(id);
    }

    @Override
    protected void load(GameResources resources){
        solid = false;
        lightLevel = 0;
        opacity = 0;
        translucent = false;
        soundPack = null;
    }
    
}

File: tests/test-minecraft-osp/src/main/kotlin/pize/tests/voxelgame/client/block/vanilla/VoidAir.kt
package pize.tests.voxelgame.client.block.vanilla

import pize.tests.voxelgame.client.block.BlockProperties
import pize.tests.voxelgame.client.resources.GameResources

class VoidAir(id: Int) : BlockProperties(id) {
    override fun load(resources: GameResources?) {
        solid = false
        lightLevel = 0
        opacity = 0
        translucent = false
        soundPack = null
    }
}
